applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -603,6 +614,25 @@ Selecting a configuration
 Before you start testing the application, you can select one of the :ref:`matter_bridge_app_custom_configs`.
 See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information how to select a configuration.
 
+.. _matter_bridge_smart_plug_functionality:
+
+Configure the functionality of the Matter-Bridge device
+-------------------------------------------------------
+
+To enable the Matter smart plug functionality, run the following command:
+
+   ``nrf54h20dk/nrf54h20/cpuapp``
+
+   .. code-block:: console
+
+      west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
+
+   ``nrf7002dk/nrf5340/cpuapp``
+
+   .. code-block:: console
+
+      west build -b nrf7002dk/nrf5340/cpuapp -p -- -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
```

Comment:
Wouldn't it be better to move it into tabs?
And instead of using a board target as a title, to just say (similarly to what we did in samples):
nRF54 DKs   |  nRF(other) DKs, but listed, e.g., nRF52 and 53 DKs?